conda/recipes/pynvjitlink/meta.yaml

```
@@ -0,0 +1,55 @@
+# Copyright (c) 2018-2024, NVIDIA CORPORATION.
+
+{% set data = load_file_data("pyproject.toml") %}
```

Let's assume all these fields exist. I would rather fail than embed empty versions (or any other metadata).
```suggestion
{% set data = load_file_data("pyproject.toml") %}
{% set project_data = data["project"] %}
{% set project_urls = project_data["urls"] %}
```

---

conda-build fails with this change:
```
Error: Failed to render jinja template in /pynvjitlink/conda/recipes/pynvjitlink/meta.yaml:
Encountered unknown tag 'project_data'.
```

We really do have to use the `.get(...).get(...)` as far as I can tell.

---

Yeah that was what we did [in Dask-CUDA]( https://github.com/rapidsai/dask-cuda/blob/d565a17f0c255eca5b9d3d8493fb26d2c77aedc1/conda/recipes/dask-cuda/meta.yaml#L49-L58 )

---

This worked fine for me locally. Perhaps some typo in how you copied it down? I've pushed my changes.